src/react/components/styleform/LineTab.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { SvgRadioButton } from 'oskari-ui';
+import { Form, Row } from 'antd';
+
+export const LineTab = (props) => {
+    return (
+        <>
+            <Row>
+                <Form.Item name='lineStyle' label='Line dash' { ...props.formLayout }>
+                    <SvgRadioButton
+                        options={ props.lineIcons.lineDash }
```

Comment:
We can bake in the options for all of these in this component. No need to pass them from the parent.